src/ServiceControl/SagaAudit/SagaAuditing.cs

```diff
@@ -112,6 +129,33 @@ public override void Enrich(IReadOnlyDictionary<string, string> headers, IDictio
                 }
 
             }
+
+            static IEnumerable<string> SplitInvokedSagas(string sagasInvokedRaw)
```

Comment:
Looking at the plugin code, the format will ALWAYS include at least 1 semicolon if there is >= 1 duplicate, correct?